### PR TITLE
Some housekeeping tasks

### PR DESCRIPTION
**Pull request by smonn**
- Fix for Husky and pre-commit hook
- Ensure we're on Node v16 across the board

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -84,7 +84,7 @@
     "wait-on": "6.0.0"
   },
   "lint-staged": {
-    "*.{ts,tsx,js,jsx}": "npm run lint -- --fix",
-    "*.{ts,tsx,js,jsx,scss,css,md}": "npm run format"
+    "*.{ts,tsx,js,jsx}": "eslint --cache --fix",
+    "*.{js,css,md}": "prettier --write"
```
Comment (smonn) [inline package.json:88]: Not 100% sure about the Prettier line. ESLint does run Prettier via `eslint-config-prettier`, so this _should_ work.

**Review by IanPirro — APPROVED** [review]
👏

File: apps/web/src/components/collectibles/collectible-browser.tsx [hunk 2]
```diff
@@ -4,7 +4,7 @@ import clsx from 'clsx'
 import Image from 'next/image'
 import { useRouter } from 'next/router'
 import useTranslation from 'next-translate/useTranslation'
-import { useState, useCallback, useEffect } from 'react'
+import { useCallback, useEffect,useState } from 'react'
```
Comment (dallashuggins) [inline apps/web/src/components/collectibles/collectible-browser.tsx:7]:
```suggestion
import { useCallback, useEffect, useState } from 'react'
```

**Review by dallashuggins — APPROVED** [review]
🙌 💯

File: apps/web/src/components/collectibles/collectible-browser.tsx (hunk 2 above)
Comment (smonn) [reply apps/web/src/components/collectibles/collectible-browser.tsx:7]: I guess Prettier didn't do its thing 🤔

**Review by afraser — APPROVED** [review]
Hot damn!

**Pull request merged**